sorted_rating.sh

```diff
@@ -0,0 +1,30 @@
+echo enter directory which contains movie folder
+read dir
+final_rate.txt
```

Comment:
What is this line? This is just a file name and nothing else. Seems like useless to me. Is it required?